python/cudf/cudf/tests/test_series.py

```diff
@@ -692,6 +692,13 @@ def test_series_round(arr, decimals):
     np.array_equal(ser.nullmask.to_array(), result.to_array())
 
 
+def test_series_round_half_up():
+    s = cudf.Series([0.0, 1.0, 1.2, 1.7, 0.5, 1.5, 2.5, None])
+    expect = cudf.Series([0.0, 1.0, 1.0, 2.0, 1.0, 2.0, 3.0, None])
+    got = s.round(how="half_up")
```

Comment:
nit: test "half_even" too?

Reply:
half_even is tested by existing test cases

Reply:
Oops, just saw the PR is just about `half_up`. This is good.